tests/resources/test_api_keys.py

```diff
@@ -0,0 +1,65 @@
+import datetime as dt
+
+import pytest
+
+from cuenca import ApiKey
+
+
+@pytest.mark.vcr
+def test_api_keys_create():
+    api_key = ApiKey.create()
+    assert api_key.id is not None
+    assert api_key.secret is not None
+    assert api_key.active
+
+
+@pytest.mark.vcr
+def test_api_keys_retrieve():
+    id_key = 'PKUvRwK7imQK2JcjJV91iEzg=='
+    api_key: ApiKey = ApiKey.retrieve(id_key)
+    assert api_key.id == id_key
+    assert api_key.secret == '********'
+
+
+@pytest.mark.vcr
+def test_api_key_deactivate():
+    id_key = 'PKyyRnEL0XS6iHeSi2_8DDPA=='
```

Comment:
@felipaoo @rogelioLpz this should most definitely not be hard-coded. Please find another way to do it